sdk/src/main/java/io/matchmore/sdk/monitoring/MatchMonitor.kt

```diff
@@ -0,0 +1,51 @@
+package io.matchmore.sdk.monitoring
+
+import io.matchmore.sdk.AlpsManager
+import io.matchmore.sdk.api.async
+import io.matchmore.sdk.api.models.Device
+import io.matchmore.sdk.api.models.Match
+import java.util.*
+
+interface MatchMonitorDelegate {
+    fun didReceiveMatches(matches: Set<Match>, forDevice: Device)
+}
+
+class MatchMonitor(private val manager: AlpsManager, private val delegate: MatchMonitorDelegate) {
+    var monitoredDevices = mutableSetOf<Device>()
+    var deliveredMatches = mutableSetOf<Match>()
```

Comment:
Can be private ?

Reply:
I want them both to be public. Devices will be set, and delivered matches are for a case where you want your all matches (not only recent ones)